data/actionsmap/yunohost.yml

```diff
@@ -288,6 +288,21 @@ user:
                         key:
                             help: The key to be removed
 
+                ### user_ssh_enable_permission()
+                enable-permission:
```

Comment:
Eh, I'm confused about : 
- the naming : if the purpose is to enable sftp, why call this `enable-permission` and not `enable-sftp` ?
- from what I understand from the PR description, the purpose it to enable sftp per user .. but this does something on a global basis - so is the purpose to enable it per user, or globally ...?

Reply:
No, by default we keep sftp enabled as it's the case now. We just add the possibility define which user/group is allowed to access by sftp.

Reply:
But this doesn't take any 'user' argument so in what sense is it per user/group ? To me this look just like a toggle on/off to enable or not SFTP ? (And therefore makes me wonder if it shouldn't just be a setting instead ?)

Reply:
Well, 

Maybe I forgot to explain this.

When you enable the permission, you have a new permission named `main.sftp`.and after you will manage the permission by the permission command see #585 .